include/interfaces/non_conservative/navier_stokes.h

```diff
@@ -256,6 +275,12 @@ NavierStokes<dim>::
 parse_parameters_call_back ()
 {
   NonConservativeInterface<dim,dim,dim+1, NavierStokes<dim> >::parse_parameters_call_back();
+
+  this->prm.leave_subsection();
```

Comment:
qui secondo me non ci va nessun `leave_subsection`.  Non sei entrato in nessuna subsection, perché dovresti uscirne?